### ISSUE TITLE
It will disable `C-x r **`

### ISSUE THREAD

**Issue by c02y** [mickeynp/discover.el#14]
A lot of commands bound to `C-x r` like `rectangle` and `bookmark` and `register`, if I use discover, all of them will not work.

BTW, [guide-key](https://github.com/kai2nenobu/guide-key) is great.

**Comment by yveszoundi** [comment]
+1 

I believe that many users don't expect those keybindings to just `vanish` out of the blue. The worst thing is that the first time it happens, you have no idea about what could be wrong.

Also see this [related reddit question](http://www.reddit.com/r/emacs/comments/2sbxun/help_undotreemode_eating_bindings/).

This issue is also kind of a duplicate of issue #7 .

**Comment by mickeynp** [comment]
The problem is discover.el is a curated list of keys. I need to re-use the guide keys implementation so it'll discover uncategorized keys and add them also. But patches welcome :)

**Comment by PhilHudson** [comment]
+1, took too much valuable time to track down the culprit and uninstall it. As Yves said, making most of the keymap vanish silently is poor. Pity, it looked great at first. Will keep watching for updates.

**Comment by mickeynp** [comment]
Yes it's a common problem. I'm half tempted to rewrite discover.el on top of guidekey. Thanks for the update, Phil.

**Comment by tarsius** [comment]
I would suggest using @abo-abo's hydra instead.

**Comment by mickeynp** [comment]
Yes! That's another good choice.
